brave/brave-core
# Toolbar Tor profile icon

Fix https://github.com/brave/brave-browser/issues/1384

![image](https://user-images.githubusercontent.com/741836/46330750-6a75bf00-c5c9-11e8-947a-7976d6cb4f06.png)

Added two sizes of vector icons since it's text and wanted to scale appropriately (the icon displays at different sizes for touch UI vs normal)

## Submitter Checklist:


## Test Plan:
- Open Tor Window
- Observe that Avatar icon is 'Tor' icon

## Reviewer Checklist:
